dhcp/install.sh

```diff
@@ -2,12 +2,33 @@
 
 source /vagrant/settings.sh
 
-#installation
+# installation
 apt --yes update
 apt --yes --install-recommends install dnsmasq
+export DEBIAN_FRONTEND=noninteractive
```

Comment:
What happens if you remove (or comment out) this line?

Reply:
This line is required to handle interactive post-install configuration. If we remove this we won't be able to handle that with our script. As a result we won't be able to install `iptables-persistent`

Reply:
> This line is required to handle interactive post-install configuration. If we remove this we won't be able to handle that with our script. As a result we won't be able to install iptables-persistent

Have you tried this?

Reply:
yes. I can share a screenshot of what happens if I remove that line. The provision script just freezes - 
![sa](https://user-images.githubusercontent.com/27950739/40978631-82047cea-68f1-11e8-8f19-ee51b756a8ad.jpg)

Reply:
> yes. I can share a screenshot of what happens if I remove that line.

If I try to remove it and there is no problem, then **you** are going to have problems. OK?

Reply:
Sure. I can make things easy for you. I have created a video which has both cases(line commented and line not commented) https://youtu.be/WQHwKu9ebYA

Reply:
Ok, do it like this:
```
DEBIAN_FRONTEND=noninteractive apt --yes install iptables-persistent
```